packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -705,7 +707,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private readonly summarizerNode: IRootSummarizerNodeWithGC;
 
     private _orderSequentiallyCalls: number = 0;
-    private _flushMode = FlushMode.Automatic;
+    private _flushMode = this.defaultFlushMode;
```

Comment:
Not big fan of calling methods on partially initialized object, even if JS magically solves problems of initialization order. Please change defaultFlushMode to be static

Reply:
will do